apps/docs/content/_data/components.json

```
@@ -292,7 +292,6 @@
 		"hideGuidelines": "false",
```

As `libs/components/src/lib/menu-item/GUIDELINES.md` is empty now, it's worth setting hideGuidelines to true:
```suggestion
		"hideGuidelines": "true",
```